### ISSUE TITLE
trailing-comma-on-call-site triggered by single argument split across multiple lines

### ISSUE THREAD

**Issue by OliverO2** [pinterest/ktlint#1820]
## Expected Behavior
The [Coding conventions](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) state (emphasis added):
> A trailing comma is a comma symbol after the last item **of a series of elements**:

So a single argument, even is split across lines, should never trigger the trailing comma rule.

## Observed Behavior
Using the code snipped below, Ktlint reports:
```
Missing trailing comma before ")" (trailing-comma-on-call-site)
```

## Steps to Reproduce
```kotlin
println(
    "This string argument is split" +
            " across multiple lines"
)
```

## Your Environment

* Version of ktlint used: 0.48.2

**Comment by paul-dingemans** [comment]
> ## Expected Behavior
> The [Coding conventions](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) state (emphasis added):
> 
> > A trailing comma is a comma symbol after the last item **of a series of elements**:
> 
> So a single argument, even is split across lines, should never trigger the trailing comma rule.

I am not a native speaker. But to me a series of things does not imply that there need to be at least two elements. Opinions on the internet seem to vary on it. Ktlint interprets a series as being at least 1 item.

The trailing comma in your example does not result in a compilation error. Even in case the function being called has declared exactly one parameter, a trailing comma on the call site does not result in a compilation error. 

The argument about a cleaner diff when adding another argument below the trailing comma are still valid. More importantly, from the perspective of KtLint the code sample below is identical to your code sample:
```
listOf(
    "This string argument is split" +
            " across multiple lines"
)```
On call site, Ktlint can not deterministicly determine what the declaration type is from the parameter in function `println`, `listOf` or `someRandomFunctionImportedFromSomewhere`.

**Comment by OliverO2** [comment]
My statement above contained a typo: "even is split" should have read "even if split".

Other than that, I'd say the most common understanding of "series" would be "more than one item".

To me, a comma at the end of a single-argument call looks so awkward that I'd trade forced consistency for more flexibility in this case. For now, I have turned off trailing commas in my projects, although I'd prefer them in cases where there are multiple arguments of the same type (e.g. a multi-line `listOf("one", "two", "three")`).

Anyway, thanks for considering.
